services/monarch-link-app/app.py

```diff
@@ -161,9 +175,12 @@ def display(id):
             try:
                 response = Response()
                 response.mimetype = "application/json"
-                response.set_data(json.dumps({"renderings": [
+                response_val = {"renderings": [
```

Comment:
Non-blocking, but is there a reason to have this renderings array that will always be one element?

Reply:
It just makes the schema consistent with what is expected from the tactile-svg handlers on the client end.

Reply:
I had forgotten to remove the redirection to the separate file! That was for testing convenience. 
I've made a new commit to fix that